cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -714,7 +714,7 @@ arrow::Status VeloxShuffleWriter::SplitListArray(const velox::RowVector& rv) {
 
     // TODO: rethink the cost of `exportToArrow+ImportArray`
     ArrowArray arrowArray;
-    velox::exportToArrow(column, arrowArray, GetDefaultWrappedVeloxMemoryPool().get());
+    velox::exportToArrow(column, arrowArray, GetDefaultLeafWrappedVeloxMemoryPool().get());
```

Comment:
Add the issue https://github.com/oap-project/gluten/issues/1194 to here, the memory is not tracked